README.md

```diff
@@ -1,159 +1,171 @@
-# Welcome\! 
-#### This repository contains the source code for:
+# Welcome to the Windows Terminal, Console and Command-Line repo
 
-  * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
-  * The Windows console host (`conhost.exe`)
-  * Components shared between the two projects
-  * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
-  * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
-#### Other related repositories include:
-  * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
+This repository contains the source code for:
 
-## Installation
+* [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
+* The Windows console host (`conhost.exe`)
+* Components shared between the two projects
+* [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
+* [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
 
-_(Note: in order to run the Windows Terminal, you'll need to be running at least Windows build 18362 or higher.)_
+Related repositories include:
 
-### Microsoft Store
+* [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
+* [Cascadia Code Font](https://github.com/Microsoft/Cascadia-Code)
 
-Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated. Or, feel free to side-load [releases](https://github.com/microsoft/terminal/releases) from GitHub, but note they won't auto-update.
+## Installing & running Windows Terminal
 
-<a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'><img src='https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.png' alt='English badge' width="284" height="104" style='width: 284px; height: 104px;'/></a>
+> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
 
-### Chocolatey (Unofficial)
+### From the Microsoft Store
 
-Download and upgrade the Windows Terminal from [Chocolatey](https://chocolatey.org).
+Install the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated:
+
+<a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'>
```

Comment:
Could future-proof this by using `https://aka.ms/install-terminal` as the link?

Reply:
Thanks for the suggestion Martin. May consider in the future.